packages/react-components/react-context-selector/package.json

```diff
@@ -36,7 +36,7 @@
     "@types/react-dom": ">=16.9.0 <19.0.0",
     "react": ">=16.14.0 <19.0.0",
     "react-dom": ">=16.14.0 <19.0.0",
-    "scheduler": "^0.19.0 || ^0.20.0"
+    "scheduler": ">=0.19.0 <=0.23.0"
```

Comment:
so this range needs to be in this specific format because invalid yarn-strict resolution ?

Reply:
@Hotell nope. This range matches specific React versions as `scheduler`. I can do `>=0.19.0`, but it will allow anything...

Reply:
As we bump React versions in requirements manually and the next version will be 19, I would postpone until it appears again.